### DISCUSSION TITLE
Option to Increase Draw Distance?

### DISCUSSION THREAD

**Discussion by wiikid6** [jorio/Nanosaur#39] [Ideas]
Hey there!

First I wanted to say thank you so much for porting this classic to modern hardware!

I was wondering, however, if you could perhaps have an option to increase the draw distance if it's at all possible within the engine? As much nostalgia as it is with the fog, the draw distance is something I think would really improve the gameplay.

Thanks!

**Comment by jorio** [comment]
Hey, here's how to do it in the code: https://github.com/jorio/Nanosaur/issues/22

I'm on the fence about just shipping this in the game because it alters the gameplay significantly. Extending the draw distance is one thing, but then the enemy spawns have to be tweaked, too. Doing it right would require thorough playtesting.

**Reply by wiikid6** [reply]
Ah thanks! Yeah, after I posted this I immediately plated it and realized this haha
